src/modules/metadata-signing.ts

```diff
@@ -0,0 +1,2 @@
+
+export const signingQueue = "metadata-signging"
```

Comment:
Move this to an environment variable?

Reply:
It's better if these don't change without a code update. The assertQueue method creates a queue if one doesn't exist by that name. Don't really want them changed too often.